apps/dashboard/app/apps/ood_app.rb

```diff
@@ -287,6 +287,14 @@ def run_setup_production
             msg += "for user #{Etc.getpwuid.name} with output: #{output}"
             raise SetupScriptFailed, msg
           end
+        else
+          msg = "Per user setup failed: file 'bin/setup-production'"
+          if !File.exist?(setup)
+            msg += " was not found at runtime."
+          elsif !File.executable?(setup)
+            msg += " is not executable. Ensure the user has access to this file."
+          end
+          raise SetupScriptFailed, msg
```

Comment:
How can this `raise` ever get executed? What's more is I wonder if we should have it at all. Seems like not.

Reply:
Yes I think we should raise the error, but I think it also needs to check if it exists. I'm not 100% sure, but I would guess `executable?` returns false if it doesn't exist. 

So it's more like `elsif File.exist?(setup) && !File.executable?(setup)`.

Also there's no need to put this back into draft. Or at least I don't mind if PRs are kept open while the work continues.